src/com/schlevoigt/JVerein/gui/control/BuchungsTextKorrekturControl.java

```diff
@@ -70,7 +70,7 @@ public Part getBuchungsList() throws RemoteException {
 		// Buchungen holen
 		query = new BuchungsKorrekturQuery();
 		if (buchungsList == null) {
-			buchungsList = new TablePart(query.get(), new BuchungAction(false));
+			buchungsList = new TablePart(query.get(), null);
```

Comment:
Warum hast du hier die Action entfernt? Ich finde es eigentlich ganz hilfreich, dass man die Buchung öffnen kann und dann ggfs. den Text selbst korrigieren kann. Bei mir gibt es Fälle wo die automatische Korrektur nicht funktioniert.

Reply:
Das Problem war, dass der Code in BuchungAction() wenn nichts ausgewählt war eine neue Buchung erzeugt hat. Da konnte ich nicht steuern welchen Filter ich für die Konten nehmen soll. Auch macht das hier keinen Sinn.
Ich habe aber später das erzeugen einer neuen Buchung aus der BuchungAction gelöscht weil es bei mir unter Wayland öfters mal vorkommt, dass zwar ein Eintrag markiert scheint aber doch die Meldung kommt, dass nichts selektiert wurde. Wenn keine Buchung selktiert wurde möchte ich nicht atomatisch eine neue erzeugen.
Weil jetzt BuchungAction nur noch editiert und keine neue erzeugt kann ich die Action wieder einbauen. Das mache ich gleich. Für das Neu erzeugen gibt es ja die BuchungNeuAction.